scripts/README.md

```diff
@@ -0,0 +1,29 @@
+# How to build `iroha` using docker
+
+# Git repository
+
+Clone `iroha` repository on your directory.
+
+```bash
+git clone https://github.com/hyperledger/iroha /path/to/iroha
+```
+
+# Development environment
+
+Build on docker container for develop environment `hyperledger/iroha-docker-develop` so you need to build it:
+
+```bash
+docker build -t hyperledger/iroha-docker-develop /path/to/iroha/docker/develop
+```
+
+# How to run
+
+```bash
+/path/to/iroha/scripts/run-iroha.dev.sh
```

Comment:
`sh /path/to/iroha/scripts/run-iroha.dev.sh` or `./path/to/iroha/scripts/run-iroha.dev.sh`